tests/extension/oneapi_device_global/device_global_exceptions.cpp

```diff
@@ -0,0 +1,289 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for the exceptions that are thrown by sycl::queue and
+//  sycl::handler copy methods overloads
```

Comment:
In C++ we say `member functions` instead of `methods`.

Reply:
Fixed